app/javascript/src/components/Dashboard/Account/Passwords/Edit.jsx

```diff
@@ -1,9 +1,41 @@
-import React, { useState } from "react";
+import React from "react";
+import * as Yup from "yup";
+import { Form, Formik } from "formik";
+import { Input as FormikInput } from "neetoui/formik";
+import { Button } from "neetoui";
+
+import registrationsApi from "apis/registrations";
 
 const Edit = () => {
-  const [currentPassword, setCurrentPassword] = useState("");
-  const [password, setPassword] = useState("");
-  const [passwordConfirmation, setPasswordConfirmation] = useState("");
+  const validationSchema = Yup.object({
+    currentPassword: Yup.string().required("Current Password is required"),
```

Comment:
Use sentence case and only capitalize words where it makes sense to: names, places etc.
```suggestion
    currentPassword: Yup.string().required("Current password is required"),
```